charts/sourcegraph-migrator/values.yaml

```diff
@@ -0,0 +1,113 @@
+# These values are dervied from https://github.com/sourcegraph/deploy-sourcegraph-helm/blob/main/charts/sourcegraph/values.yaml
+# 🚨 This file MUST be kept in sync with the values definition in sourcegraph/values.yaml
+# 🚨 This file must be compatible with sourcegraph/values.yaml
+
+### Common Configuration ###
+sourcegraph:
+  affinity: {}
+  image:
+    defaultTag: '{{ .Chart.AppVersion }}'
+    pullPolicy: IfNotPresent
+    repository: index.docker.io/sourcegraph
+    # When set to true, sourcegraph.image.defaultTag is used as the default defaultTag for all services, instead of service-specific default defaultTags
+    useGlobalTagAsDefault: false
+  # Mount a named secret containing docker credentials
+  imagePullSecrets: []
+  # Add a global label to all resources
+  labels: {}
+  # Set a custom name for the app.kubernetes.io/name annotation
+  nameOverride: ""
+  nodeSelector: {}
+  podAnnotations: {}
+  podLabels: {}
+  tolerations: {}
+
+
+# Generic application configuration options, used by most applications below
+# app: # Generally matches directory name
+#   replicaCount: 1
+#
+#   image:
+#     # Application-specific default tag. Overrides the global tag. Do not modify - use tag instead
+#     defaultTag:
+#     # Override the default tag on a service-specific basis
+#     tag:
+#     # Override the default docker image name
+#     name:
+#
+#   # Add additional labels and annotations to various resources
+#   labels: {}
+#   podLabels: {}
+#   podAnnotations: {}
+#   serviceLabels: {}
+#   serviceAnnotations: {}
+#   # Override default service/deployment names
+#   name: ""
+#   # Provide custom environment variables
+#   env: {}
+#   # Set resource requests / limits
+#   resources: {}
+#   podSecurityContext: {}
+#   securityContext: {}
+#   # Add additional volumes and mounts to deployments
+#   extraVolumeMounts: {}
+#   extraVolumes: {}
+#   # Add sidecars
+#   extraContainers: {}
+#   # Toggle serviceAccount creation
+#   serviceAccount.create: true
+#   # Customize service account name
+#   serviceAccount.name: ""
+#   # Override service type (defaults to ClusterIP)
+#   serviceType: ""
+#   # Set disk size on PVC's for stateful deployments
+#   storageSize: ""
+#   # Optionally bind PVC's to an existing PV
+#   volumeName: ""
+#   # Use an existing configmap (specified by name) instead of creating one
+#   existingConfig: ""
+#   # Toggle deployment of applications on/off. Applies to databases and third-party applications
+#   enabled: true
+
+frontend:
```

Comment:
The mixture of frontend and migrator env's feels messy here - the migrator really is a standalone job here, not associated with the frontend at all.

Reply:
@caugustus-sourcegraph 

Yeah, I agree a mix of frontend and migrator values is really confusing.

I will just move all frontend values to migrator, and we can document that clients can reuse the env var for frontend in migrator.